components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -233,12 +233,11 @@ ads::CreativeAdNotificationList GetCreativeAdNotificationsOnFileTaskRunner(
 ads::AdConversionList GetAdConversionsOnFileTaskRunner(
     const std::string& url,
     BundleStateDatabase* backend) {
-  ads::AdConversionList ad_conversions;
-
   if (!backend) {
-    return ad_conversions;
+    return {};
   }
 
+  ads::AdConversionList ad_conversions;
```

Comment:
If you rely on `copy elision`, then you should leave `ad_conversions` where it was initially and make sure it's returned on all paths instead of returning `initializer_list`.

Reply:
Fixed